iam.go

```diff
@@ -57,6 +59,13 @@ type Client struct {
 
 	// S3Credentials instance is used to make requests against Selectel IAM API and manage S3 Credentials.
 	S3Credentials *s3credentials.Service
+
+	// Federations instance is used to make requests against Selectel IAM API and manage Federations.
+	Federations *saml.Service
+
+	// FederationsCertificates instance is used to make requests against Selectel IAM API
+	// and manage Federations Certificates.
+	FederationsCertificates *certificates.Service
```

Comment:
As `FederationsCertificates` are used for `Federations` object, maybe it would be better to inject `FederationsCertificates` client into `Federations`.

Reply:
Moved `Certificate` instance inside of `SAMLFederations`